README.md

```diff
@@ -1649,6 +1649,7 @@ Software and tools for managing [recipes](https://en.wikipedia.org/wiki/Recipe).
 - [kcal](https://github.com/kcal-app/kcal) - Track nutritional information about foods and recipes, set goals, and record a food journal to help along the way. Kcal is a personal system that focuses on direct control of inputs and a minimal, easy to use recipe presentation for preparing meals. ([Demo](http://demo.kcal.cooking/login)) `MPL-2.0` `PHP`
 - [Mealie](https://hay-kot.github.io/mealie/) - Material design inspired recipe manager with category and tag management, shopping-lists, meal-planner, and site customizations. Mealie is focused on simple user interactions to keep the whole family using the app. ([Demo](https://mealie-demo.hay-kot.dev/), [Source Code](https://github.com/hay-kot/mealie)) `MIT` `Python`
 - [RecipeSage](https://github.com/julianpoy/recipesage) - A recipe keeper, meal plan organizer, and shopping list manager that can import recipes directly from any URL. ([Demo](https://recipesage.com)) `AGPL-3.0` `Nodejs`
+- [Recepturer](https://www.recepturer.com/) - Let Recepturer help you organize all your recipes in one place. You can add, edit and delete recipes, and create meal plans. Simply type your recipe and all ingredients will be listed as you go. Data is stored on Dropbox. ([Demo](https://recepturer.com/), [Source Code](https://github.com/sjoerdvanderhoorn/recepturer)) `MIT` `Javascript`
```

Comment:
> https://github.com/sjoerdvanderhoorn/recepturer#installation

Installation docs could be written better so that users know how to self host on their own machine,

- Please remove the extra demo link: `[Demo](https://recepturer.com/

Reply:
Thank you, addressed both suggestions.